chia/simulator/start_simulator.py

```diff
@@ -46,13 +52,14 @@ def service_kwargs_for_full_node_simulator(root_path: Path, config: Dict, bt: Bl
         service_name=SERVICE_NAME,
         server_listen_ports=[service_config["port"]],
         on_connect_callback=node.on_connect,
-        rpc_info=(SimulatorFullNodeRpcApi, service_config["rpc_port"]),
```

Comment:
I assume this just has been moved to match the other files?

Reply:
Sure, let's go with that.  I don't recall but it makes sense.